Client/src/ArmoniK.DevelopmentKit.Client.Common/Submitter/BaseClientSubmitter.cs

```diff
@@ -568,36 +507,39 @@ public byte[] GetResult(string taskId, CancellationToken cancellationToken = def
         }
       }
 
-      return result.ToArray();
+      return ms.ToArray();
     }
 
 
     /// <summary>
     ///   Try to find the result of One task. If there no result, the function return byte[0]
     /// </summary>
-    /// <param name="taskId">The task Id trying to get result</param>
+    /// <param name="resultId">The Id of the result</param>
     /// <param name="checkOutput"></param>
     /// <param name="cancellationToken">The optional cancellationToken</param>
     /// <returns>Returns the result or byte[0] if there no result or null if task is not yet ready</returns>
     [UsedImplicitly]
```

Comment:
FYI : I think I'd be better to specify `[PublicApi]` rather than `[UsedImplicitely]`. The latter is best fitted for reflection use cases.